### ISSUE TITLE
The new code doesn't follow redirects

### ISSUE THREAD

**Issue by bugbaba** [Damian89/extended-xss-search#5]
Hi Damian,

I have been using the old XSSFinder code for a quite a bit and one of its feature is that it follows the http [redirects](https://github.com/Damian89/xssfinder/blob/master/inc/requests.py#L77) and also checks for them and [forward the request](https://github.com/Damian89/xssfinder/blob/master/inc/requests.py#L22)

I have been trying this new code and seems to be this feature is missing from this new code.
you can reproduce:
by setting `http://forums.sonyinsider.com/topic/29805-sony-annual-report-1992/page/?xyz=lolwa` as url and observe the traffic in burp it never follows the redirect.

![image](https://user-images.githubusercontent.com/29960776/62427215-25507b00-b70d-11e9-890b-f8dfeb96066f.png)

Thanks for this lovely code :heart:

**Comment by Damian89** [comment]
Yeah you are right, will fix it by time (or maybe someone else is faster) - currently I have a lot other stuff to do, so my time on open source stuff is very limited!
